src/Entities/Behaviors/HasUpdatedDate.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Choredo\Entities\Behaviors;
+
+trait HasUpdatedDate
+{
+    /**
+     * @ORM\Column(
+     *     type="datetime",
+     *     name="updated",
+     *     nullable=false
+     * )
+     *
+     * @var \DateTime
+     */
+    private $updatedDate;
+
+    /** @ORM\PrePersist */
+    public function updateUpdatedDate()
+    {
+        if ($this->updatedDate === null) {
```

Comment:
This isn't accurate. Updated dates should update regardless of being previously set, no?